src/shared_modules/rsync/tests/interface/rsync_test.cpp

```diff
@@ -17,6 +17,7 @@
 #include "rsync.hpp"
 #include "dbsync.h"
 #include "dbsync.hpp"
+#include "cjsonSmartDeleter.hpp"
```

Comment:
There's an unused struct CJsonDeleter in this file https://github.com/wazuh/wazuh/blob/15018-cppcheck-static-analysis-errors/src/shared_modules/rsync/tests/interface/rsync_test.cpp#L66

Reply:
good catch!